modules/ROOT/pages/gatekeeper.adoc

```diff
@@ -47,34 +41,38 @@ anypoint.platform.gatekeeper=flexible
 
 === Security Level Definitions
 
+You can apply one of the listed security levels to Gatekeepr:
+
 ==== False
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process. This value is available only for backward compatibility in Mule 3.x. Use ‘disabled’ instead. `disabled` is not available in all 3.x releases. `false` is not available in Mule 4.x.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘disabled’ instead for all other scenarios. Note that the `disabled` value is not available in all Mule 3.x releases and the `false` value is not available in Mule 4.x releases.
 
 ==== Disabled
 
-When this value is set, no security is applied. Unwelcome traffic may reach the APIs during the policies application process.
+When you set this value, Gatekeeper does not apply any security. With this setting, the API might receive unwanted traffic during the period while the policies are being applied.
 
 ==== True
 
-When this value is set, the default security level is Strict. This value is available only for backward compatibility in Mule 3.x. Use ‘strict’ or ‘flexible’ instead. `true` is unavailable in Mule 4.x
+When you set this value, the default security level is  set to `Strict`. In Mule 3.x, this value is available only for backward compatibility. Therefore, use ‘strict’ or ‘flexible’ instead for all other scenarios. The value `true` is unavailable in Mule 4.x releases.
 
```

Comment:
I think all this narrative is complicating the point that security levels used in Mule 3.x are replaced in Mule 4.x.
It might be more clear to use a table that defines the actual security and then has a column each for value in Mule 3 and Mule 4, something like...

**Type of Security-----Definition-------------Level in Mule 3.x-----       Level in Mule 4.x-----**

None-------------------blah blah-----------------`false`------- -----------`disabled`

Reply:
Done! Thanks!